sabnzbd/interface.py

```diff
@@ -66,7 +66,7 @@
 from sabnzbd.api import list_scripts, list_cats, del_from_section, \
     api_handler, build_queue, build_status, \
     retry_job, retry_all_jobs, build_header, build_history, del_job_files, \
-    format_bytes, std_time, report, del_hist_job, Ttemplate, build_queue_header, \
+    format_bytes, report, del_hist_job, Ttemplate, build_queue_header, \
```

Comment:
You can also remove the `std_time` function, since it's not used anymore 👍

Reply:
@Safihre 👍 done